### DISCUSSION TITLE
bug: Do not respond after a `get`

### DISCUSSION THREAD

**Discussion by BobSquarePants** [rickysarraf/apt-offline#236] [Q&A]
Hi,

after `apt-offline get offlinemachine.sig --bundle pkgs.zip`

**apt-offline** stay blocked to 

> Fetching APT Data
>
> Downloading orphan-sysvinit-scripts

and do not return to the prompt....

system: [Devuan 4](https://www.devuan.org/)

**Comment by rickysarraf** [comment]
You could always run it in `--verbose` mode. That'd give some extra bit of information. On the other hand, from what you've described, this sounds like a network issue.
